src/style.css

```diff
@@ -1 +1,20 @@
 /* styles go here */
+
+.fieldset {
+  margin-top: 0;
+  margin-bottom: 20px;
+}
+
+.form_field {
+  display: block;
+  margin: 0 0 10px;
+}
+
+.form_field:last-child {
+  margin: 0;
+}
+
+.paragaph {
+  display: inline;
+  margin-top: 0;
+}
```

Comment:
I can't find a class with `paragaph` value. You'll be able to remove it.

Reply:
removed